packages/jumpstarter-driver-android/jumpstarter_driver_android/driver/emulator.py

```
@@ -0,0 +1,366 @@
+        self.logger.info(f"Starting Android emulator with command: {' '.join(cmdline)}")
+        self._process = subprocess.Popen(
+            cmdline,
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=False,  # Keep as bytes for proper encoding handling
+            env=env,
+        )
+
+        # Process logs in separate threads
+        self._log_thread = threading.Thread(target=self._process_logs, args=(self._process.stdout,), daemon=True)
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)